tests/plando/boss_rush_dc_mq.json

```diff
@@ -4,6 +4,7 @@
   "shuffle_dungeon_entrances":               true,
   "open_forest":                             "open",
   "mq_dungeons":                             1,
+  "dungeon_shortcuts_choice":                "choice",
```

Comment:
Should be `choices`

Reply:
What word should be in plural, the name of settings "dungeon_shortcuts_choices" or the name of property "choices" ?

Reply:
The name of the property, the same as it's `choices` in the other plandos

Reply:
Oh, i didn't understand, all plando with copy/paste I did a mistake, the option value is without 's' in SettingsList

Reply:
Fix : https://github.com/TestRunnerSRL/OoT-Randomizer/pull/1493/commits/ae9717c35a33e758bca64f3f9de721d194555bd9